docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -1681,6 +1682,17 @@ Moreover, don't forget to add LinkedIn as a registration mode from your portal p
 
 Also, LinkedIn requires a 'state' parameter for the authorization URL. If you modify it, make sure to add it at the end of your URL.
 
+Twitter
+^^^^^^^
+To use Twitter, you also need an API code and a secret key which Twitter calls consumer key and consumer secret.  To get these, you need to create an App here: https://apps.twitter.com/.  When you create your App, make sure you specify the following as the Callback URL
```

Comment:
Formatting suggestion:
> .. `which Twitter calls _consumer key_ and _consumer secret_.`

And I propose to rephrase the following sentence as follow:

> `Obtain this information by creating an new application from your https://apps.twitter.com/[Twitter Apps Management page].`

And finally:

> .. `the following as the _Callback URL_:`